Themes/package-lock.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
I don't think this file should be deleted, should it?

Reply:
There are no dependencies in the package.json to which to lock.  The package.json exists only to inform VS Code about the package.  Is there something this is needed for that I may not be aware of?

Reply:
If we don't need it, it can be removed.  I was questioning because it seemed to be generated at least at some point because it's here in the repo.

Reply:
I looked inside the current project-lock.json and it barely contains anything. I believe we do not need this file.

Reply:
When I run npm install in this folder, it is regenerated.  So we should either leave it alone or add it to `.gitignore` for this folder.  If we don't, it will be added back the next time someone works in this folder.

Reply:
@bobbrow Can we check this in? Maybe we can re-add the package.json in a follow up PR?